src/dashboards/components/dashboard_index/DashboardsIndex.tsx

```diff
@@ -104,7 +95,7 @@ class DashboardIndex extends PureComponent<Props, State> {
               <GetAssetLimits>
                 <DashboardsIndexContents
                   searchTerm={searchTerm}
-                  onFilterChange={this.handleFilterDashboards}
+                  onFilterChange={this.props.setSearchTerm}
```

Comment:
if the `onSearch` and `onFilterChange` are being mapped to props, there's no reason for us to prop drill. We could just map it out directly to the child